resources/MrMathTables.js

```diff
@@ -0,0 +1,2289 @@
+
```

Comment:
Can we split this into additional files e.g.
* skills.js 
* inventory.js?
* combat.js
* occupations.js
Also, since you moved this to a sub-folder, it will be missed by the campaign_publish.sh command.

Reply:
MrMathTables.js is an alternate version of tables.js for Mr Math's campaign